client/scss/components/_header_block.scss

```diff
@@ -0,0 +1,107 @@
+//* @define header-block;
+
+.header-block {
+  &.header-block--a {
+    @include respond-to('large') {
+      position: relative;
+      padding: 1.8rem 0 1.5rem span-columns('large', 1.5);
+
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: span-columns('large', 0.5);
+        background: color('keyline-grey');
+      }
+    }
+  }
+
+  &.header-block--b,
+  &.header-block--c:not(.header-block--xlarge) {
+    padding-bottom: 1.2rem;
+    border-bottom: 1px solid color('keyline-grey');
+  }
+}
+
+.header-block__title {
+  margin-top: 0;
+  margin-left: -0.06em;
+
+  .header-block--a & {
+    @include font('WB3');
+    margin-bottom: 0.25em;
+  }
+
+  .header-block--b & {
+    @include font('HNM1');
+  }
+
+  .header-block--c.header-block--small & {
```

Comment:
I know what you mean. But then I also think what's nice about having the sass componentized is that you can reason comfortably about an element within that component and not have to worry about specificity wars.

I've previously found using a parent selector to scope modified elements within it has been a decent compromise, but I'm willing to try a firmer approach if that's what we want to do.